docs/1-Getting-Started/2-Starting-new-project.md

```diff
@@ -1,12 +1,6 @@
-> We use Laravel as an example, but **kool** is a **multi-stack** tool that can be used with any language and framework. [Check out our presets](/docs/presets/introduction) for easily creating projects using other frameworks (e.g. WordPress, Symfony, Next.js, Nuxt.js, NestJS, Adonis, Hugo, etc).
+> We use PHP for our Hello World example, but **kool** is a **multi-stack** tool that can be used with any language and framework. For instance, [check out our presets](/docs/presets/introduction) for easily creating projects using JavaScript frameworks like Next.js, Nuxt.js, NestJS, and AdonisJs.
 
-## Start a New Project with Docker in 3 Easy Steps
-
-1. Run `kool create`
-2. Update **.env.example**
-3. Run `kool run setup`
-
-> Yes, using **kool** + Docker to create, work on, and switch between projects is that easy!
+## Start a New Project with Docker in 4 Easy Steps
```

Comment:
I think it is important to notice/disclaimer here that, for creating from scratch a Docker environment managed with Kool, it is obvious you need at least basic knowledge on Docker and Docker Compose, knowing your way around those tools.

If you would like a pre-defined environment on top of these tools, that is our presets, removing this learning curve.

Reply:
@fabriciojs - added disclaimer (see latest commit).